query-compiler/query-engine-tests-todo/pg/fail

```diff
@@ -245,62 +234,19 @@ queries::filters::self_relation::self_relation_filters::one2one_null
 queries::filters::self_relation::self_relation_filters::one2one_null_fail
 queries::filters::self_relation_regression::sr_regression::all_categories
 queries::order_and_pagination::nested_multi_order_pagination::paging_one2m_stable_order::cursor_child_3
-queries::order_and_pagination::nested_multi_order_pagination::paging_one2m_stable_order::take_first_child_each_parent
```

Comment:
A lot of these tests a probably fixed because they are now using much simpler query plans with more work on the database side, but the corresponding functionality is still broken for most users who aren't using the `relationJoins` preview feature. We should have separate CI jobs running with different `PRISMA_RELATION_LOAD_STRATEGY` values and separate `fail` lists.